README.md

```diff
@@ -1 +1,83 @@
-# newyorktimes-clone
\ No newline at end of file
+> newyorktimes-clone page
+
+![screenshot](./img/screenshot.png)
+
+A clone of the beautifull new york times editorial.
+
+## Built With
+
+- HTML5 and CSS3
+- VSCode, Linters, Stylelint, Validator.w3
+
+## Live Demo
+
+[Live Demo Link](https://peaceful-stonebraker-d86e1f.netlify.com/)
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- Get your browsers up to date
+
+### Setup
+
+- Clone the repository on your local machine
+- cd into the folder, find the index.html file and open it
+
+### Install
+
+- Install VSCode or any code editor you like
+- Install Node JS and nmp(most recents versions)
+- Run this command on your terminal in order to install stylelint: npm install stylelint stylelint-config-recommended --save-dev
+
+### Usage
+
+### Run tests
+
+- Test the css by typing the command: npx stylelint .
+- Go to http://validator.w3.org/ , upload the index.html file to test it.
+
+### Deployment
+
+- Get an online web hosting
+- Upload all the files in the htdocs folder of your domain
+- Access your domain name in browser
+
+## Authors
+
+👤 **Author1**
+Patrick Nyatindo
+
+- Github: [@githubhandle](https://github.com/nyatindopatrick)
+- Twitter: [@twitterhandle]()
+- Linkedin: [linkedin]()
+
+👤 **Author2**
+Aremu Lucky
+
+- Github: [@githubhandle](https://github.com/Luckyaremu)
+- Twitter: [@twitterhandle]()
+- Linkedin: [linkedin]()
+
+## 🤝 Contributing
+
+Contributions, issues and feature requests are welcome!
+
+Feel free to check the [issues page]().
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```

Comment:
This needs review. Whose code was used? Where did the inspiration come from?

 The project has a descriptive README.md ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

Reply:
![footer](https://user-images.githubusercontent.com/37473601/73246729-a0bc5200-41af-11ea-8df2-e0ed868925ab.jpg)
 "The footer line is positioned correctly, all other issues have been solved"